multiqc/utils/search_patterns.yaml

```diff
@@ -319,6 +319,10 @@ samtools/idxstats:
 samtools/rmdup:
     contents: '[bam_rmdup'
     shared: true
+sargasso:
+    fn: 'overall_filtering_summary.txt'
+fastqc/zip:
+    fn: '*_fastqc.zip'
```

Comment:
I think the fastqc addition here may be a mistake?

Reply:
yes that's a mistake